team.html

```diff
@@ -329,7 +294,7 @@ <h3>FOLLOW US</h3>
                 <div class="col-md-3">
                     <h3>SPONSOR US</h3>
                     <p>
-                        Vivien Ding<br/>
+                        Waleed Ahmed<br/>
```

Comment:
This should go to the business/marketing lead. From above it seems that it's Jason now?